structure/format.go

```diff
@@ -151,6 +152,12 @@ func (f *Formatter) outputFields(entry *Entry, raw json.RawMessage) {
 				continue
 			}
 			if !f.shouldSkipField(key, value) {
+				switch v := value.(type) {
+				case float64:
+					output = append(output, key+"="+strconv.FormatFloat(float64(v), 'f', -1, 64))
```

Comment:
When using a type switch in Go you can omit the extra cast in the case scope, so `..FormatFloat(v, ..)` will do. [ref](https://tour.golang.org/methods/16)